docs/components/BarChartDocs.js

```diff
@@ -97,13 +97,13 @@ class BarChartDocs extends React.Component {
         <h5>data<label>Array</label></h5>
         <p>An array of objects that label and value properties. Each object represents a bar. Optionally allows a color property to override the default color of the bar. Example:</p>
         <Markdown lang='js'>
-  {`
-    [{
-      label: 'label', //optional - Anything that can be parsed by d3 as an axis value.
-      color: '#E3E6E7', //string - hexcode or rgb
-      value: 10 //number - required, value of bar, determines bar size
-    }]
-  `}
+          {`
+            [{
+              label: 'label', //optional - Anything that can be parsed by d3 as an axis value.
+              color: '#E3E6E7', //string - hexcode or rgb
+              value: 10 //number - required, value of bar, determines bar size
+            }]
+          `}
```

Comment:
With the spacing in the markdown blocks it used to including the spacing in the docs and it looked weird. Can you double check that's not happening here?